site/realm/functions/onIncidentUpdate.js

```diff
@@ -14,8 +14,11 @@ exports = async function (changeEvent) {
     return;
   }
 
+  const notificationWhitelistedFields = ['title', 'description', 'date', 'Alleged deployer of AI system', 'Alleged developer of AI system', 'Alleged harmed or nearly harmed parties', 'reports', 'editors', 'editor_notes'];
+
   const updatedFields = Object.keys(updateDescription.updatedFields);
 
+  const willNotifyIncidentUpdate = updatedFields.some(field => notificationWhitelistedFields.includes(field));
```

Comment:
Why not just return here if no fields are worth notifying? (instead of letting the db queries and complicated logic below start running)
I note that we return early in the above code for "invalid" change events, so this seems fair and safe, but I could be missing a reason.

(drive-by comment 🚗 )